managed/services/telemetry/config.default.yml

```diff
@@ -267,14 +267,22 @@ telemetry:
         value: 1
 
   # PMM - Usage
-  - id: GrafanaStatActiveMonthlyUsers
+  - id: GrafanaStatMonthlyActiveUsers
     source: VM
     query: grafana_stat_active_users
-    summary: "Active monthly users"
+    summary: "Monthly active users"
     data:
       - metric_name: "pmm_server_grafana_stat_active_users"
         value: 1
 
+  - id: GrafanaStatDailyActiveUsers
+    source: GRAFANADB_SELECT
+    query: count(*) AS count FROM user WHERE last_seen_at > time('now') - 86400
```

Comment:
What about:
`WHERE date(last_seen_at) = date('now')`?

Reply:
So i use same solution as in [Grafana](https://github.com/grafana/grafana/commit/9dfd469afc8bb597c610662612e0fdb13fc2f31f#diff-484a31f371b93925d8bfa3afe2ce1775d22eb911df01aeb1f327cedbb96b4f58R58):
```go
dailyActiveUserDeadlineDate := time.Now().Add(-24*time.Hour)
	sb.Write(`(SELECT COUNT(*) FROM `+dialect.Quote("user")+` WHERE last_seen_at > ?) AS daily_active_users,`, dailyActiveUserDeadlineDate)
```

We send stats each 24 hours so i think in average will be same